js/jsmind.js

```diff
@@ -1666,6 +1669,31 @@
             for (var i = 0; i < l; i++) {
                 this.event_handles[i](type, data);
             }
+        },
+
+        // Drag the whole mind map with your mouse, when it's larger that the container
+        _drag_nodes: function () {
+            if (this.options.draggable) {
+                // Avoid scrollbars when mind map is larger than the container (e_panel = jsmind-inner)
+                this.view.e_panel.style = 'overflow: hidden'
+                // Move the whole mind map with mouse moves, while button is down.
+                this.view.container.onmousedown = (eventDown) => {
+                    // Record current mouse position.
+                    let x = eventDown.clientX
+                    let y = eventDown.clientY
+                    // Stop moving mind map once mouse button is released.
+                    this.view.container.onmouseup = () => {
+                        this.view.container.onmousemove = null
+                    }
+                    // Follow current mouse position and move mind map accordingly.
+                    this.view.container.onmousemove = (eventMove) => {
```

Comment:
it's better to avoid `onmousemove` with a flag that clear the handler

```javascript
dragging = false
function onmousedown(){
    dragging = true
}

function onmouseup(){
    dragging = false
}

functionn onmousemove(){
    if(dragging) {
        ...
    }
}
```